install/kubernetes/cilium/values.yaml

```diff
@@ -1076,6 +1076,12 @@ ipam:
 # -- Configure the eBPF-based ip-masq-agent
 ipMasqAgent:
   enabled: false
+  config:
+    nonMasqueradeCIDRs:
+    - 10.0.0.0/8
+    - 172.16.0.0/12
+    - 192.168.0.0/16
```

Comment:
I don't know if we can ship these by default. The network may or may not guarantee that bidirectional connectivity between pod IPs and these IP ranges will work. These need to be configured by the user.

Reply:
Ok, We don't need to set the default values and let the user configure.